test/unit/registration.js

```diff
@@ -589,3 +590,48 @@ exports['trigger param configuration parse failure for invalid JSON propagates t
         test.done();
     });
 };
+
+exports['addMessage prepops and passes the right information to messages.addMessage'] = (test) => {
+    const testPhone = '1234',
+          testMessage = 'A Test Message',
+          testRegistration = 'some registrations',
+          testPerson = 'a patient contact';
```

Comment:
This test mocks out `messages.getMessage` so that call is never exercised. I've just added a couple of unit tests in `test/unit/messages.js`, fresh off of my dome, so check those out